Where_Is_My_Wife/Assets/_Main/Scripts/Player/Controller/PlayerController.cs

```diff
@@ -0,0 +1,136 @@
+using System;
+using Cysharp.Threading.Tasks;
+using UniRx;
+using UnityEngine;
+using UnityEngine.Serialization;
+using WhereIsMyWife.Managers;
+using WhereIsMyWife.Player.State;
+using Zenject;
+
+namespace WhereIsMyWife.Controllers
+{
+    public partial class PlayerController : IPlayerControllerData
+    {
+        public Vector2 RigidbodyVelocity => _rigidbody2D.velocity;
+        public Vector2 GroundCheckPosition => _groundCheckTransform.position;
+        public Vector2 WallHangCheckUpPosition => _wallHangCheckUpTransform.position;
+        public Vector2 WallHangCheckDownPosition => _wallHangCheckDownTransform.position;
+        public float HorizontalScale => transform.localScale.x;
+    }
+    
+    public partial class PlayerController : MonoBehaviour
```

Comment:
Make this MonoBehhavior.

Reply:
bb978ba3e2a5ab91045a28ee45623469ec2b813d